extensions/rich_text_components/rte-output-display.component.ts

```diff
@@ -184,6 +184,18 @@ export class RteOutputDisplayComponent implements AfterViewInit {
        * cycle and hence, we will still have the problem.
        */
       this.show = false;
+      // The text node we inserted outside angulars influence is causing some
+      // leaks. So we are manually removing the text nodes we attached.
```

Comment:
Explain what "causing some leaks" means. What does the user see?

Reply:
changed